visualization/helperJSFiles/handleClicks.js

```diff
@@ -21,6 +21,14 @@ function handleClickTreeMapTypeSat(value, checked) {
         }
     }
 }
+/**
+* Handles all clicks on label redirect to a click on the box
+*/
+function onClickLabel(idToggle) {
+	console.log(idToggle);
```

Comment:
Logging all the click information is nice to keep? Not sure if you want the user to see that :stuck_out_tongue: